lib/0001_retrieve-secrets.sh

```diff
@@ -25,16 +25,17 @@ trap 'err_report $LINENO' ERR
 #}
 #'
 
-# making sure that no tracing takes place, we're dealing with secrets here :)
+# Prevent tracing to ensures secrets won't be leaked.
 declare xtrace=""
 case $- in
   (*x*) xtrace="xtrace";;
 esac
 set +x
 
-# inject secrets into environment
-pushd $1
-  eval "$(./vendor/conjur-env)"
+# $HOME points to the app directory, which should contains a secrets.yml file.
```

Comment:
Is there documentation we can link to that will share how we know HOME and DEPS_DIR will be set when this script is run?

Reply:
Unfortunately no. This is knowledge I gained from the examples linked to in Slack conversations. This seems to be quite poorly documented.